en/docs/tutorials/create-and-publish-awslambda-api.md

```diff
@@ -58,7 +58,7 @@ Now, you have created a new REST API successfully.
 
     1. Click delete, as shown below, to remove all the existing resources.
 
-          [![Delete all existing resources]({{base_path}}/assets/img/learn/delete-all-existing-resources.jpg)]({{base_path}}/assets/img/learn/delete-all-existing-resources.jpg)
+          [![Delete all existing resources]({{base_path}}/assets/img/learn/delete-all-exisiting-resources.png)]({{base_path}}/assets/img/learn/delete-all-exisiting-resources.png)
```

Comment:
Even though this path is correct please change the name of the image as there is a typo in it.

Reply:
Ok. It looks like the same image is linked to multiple places. For now shall we merge this? I'll send in a fresh PR with the fix.